app/uk/gov/hmrc/traderservices/views/StartView.scala.html

```diff
@@ -34,7 +34,7 @@
     )
 
     @html.p {
-        @html.a("/trader-services/route-one", messages("view.start.route-one.link.text"))
+        @html.a("/trader-services/pre-clearance", messages("view.start.route-one.link.text"))
```

Comment:
According to the **content spreadsheet** we want: 
**/trader-services  TO BECOME    /pre-clearance-check-route-1**

(We might want to discuss this with Irene though as it might be better to move away from calling everything route 1, as discussed yesterday) 

**/consignment-details  TO BECOME  /declaration-details**
This also applies to the heading (What are the declaration details?)